code/modules/antagonists/monkey/monkey.dm

```diff
@@ -164,6 +164,17 @@
 			return TRUE
 	return FALSE
 
+/datum/team/monkey/proc/infected_gorillas_alive()
```

Comment:
Unused?

Reply:
no, see, monky has a proc for each type of animal. This especially needs its own proc, because rabid gorillas dont actually count as having the disease due to being simplemobs.